setup.py

```diff
@@ -26,23 +26,25 @@
 REQUIREMENTS = [
     "fs>=2.0.0",
```

Comment:
This should probably be `"fs~=2.0.8"` to only get backwards incompatible releases greater than 2.0.8. I'm sticking to semver, so any backwards incompatible changes will reset `fs` to  `3.0.0`.